src/ui/ui.css

```diff
@@ -31,11 +31,11 @@ body {
   animation: mspin-rotate 1568.63ms infinite linear
 }
 
```

Comment:
Need to add `swg-container {display: block}`. This is a general rule of thumb: as soon as you replace `div` with a `swg-x`, immediately add `swg-x {display:block}`. It'd be even better to combine them all together:

```
.swg-container, .swg-loading, .swg-x, .swg-y, .swg-etc {display: block;}
```

Reply:
Done